exercises/practice/killer-sudoku-helper/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+A friend of yours is learning how to solve Killer Sudokus but struggling to figure out which digits can go in a cage.
+They ask you to help them out by writing a small program that lists all valid combinations for a given cage, and any constraints that affect the cage.
+
+To make the output of your program easy to read, the combinations it returns must be sorted.
+
+## Killer Sudoku Rules
+
+- [Standard Sudoku rules](https://en.wikipedia.org/wiki/Sudoku) apply.
```

Comment:
Not blocking, but I'll put up a one-page website with the rules of Sudoku on to make it easy to understand (without reading the whole wikipedia page, or having to go to some site that makes me accept a cookie banner!)

Reply:
I could add them here too, I guess?

Reply:
From an not-sudoku player, the pictures definitely help.  😄   I agree with @iHiD tho -- a one-pager with a clear ruleset might help as well.  The wiki article is clear....but there is a lot of detail there.

Reply:
...and having a reference/explanation of the differences might also help:  https://en.wikipedia.org/wiki/Killer_sudoku.

Reply:
I don't like linking to Wikipedia pages that start with a big "This article has multiple issues" warning but I'll try to find a resource/tutorial on killer sudokus.

Reply:
Yeah.  I was going to comment on that warning at the top...but I figured that there was general "research" for directions needed.  😛

Reply:
```suggestion
- [Standard Sudoku rules](https://masteringsudoku.com/sudoku-rules-beginners/) apply.
```